src/main/java/me/zeroeightsix/kami/module/modules/misc/AutoRespawn.java

```diff
@@ -11,24 +11,31 @@
 
 /**
  * Created by 086 on 9/04/2018.
+ * Updated 16 November 2019 by hub
  */
-@Module.Info(name = "AutoRespawn", description = "Automatically respawns upon death and tells you where you died", category = Module.Category.MISC)
+@Module.Info(name = "AutoRespawn", description = "Respawn utility", category = Module.Category.MISC)
 public class AutoRespawn extends Module {
 
-    private Setting<Boolean> deathCoords = register(Settings.b("DeathCoords", false));
     private Setting<Boolean> respawn = register(Settings.b("Respawn", true));
+    private Setting<Boolean> deathCoords = register(Settings.b("DeathCoords", false));
+    private Setting<Boolean> antiBug = register(Settings.b("Anti Bug", true));
```

Comment:
`Anti Bug` isn't very descriptive. Maybe mention the screen - e.g. `Anti Glitch Screen`?

Reply:
Oh yeah yo are right. I changed it [here](https://github.com/zeroeightysix/KAMI/pull/140/commits/9bc05ebac8aae252cb21c6c314a0e16259892636).